components/bitcoin-topics/Navigation.tsx

```diff
@@ -117,35 +179,66 @@ export function Navigation({
-                                        onClick={onLinkClick}
+                                    <div
                                         className={clsx(
-                                            "block font-medium w-full pl-3.5 before:pointer-events-none before:absolute before:-left-1 before:top-[0.5em] before:h-1.5 before:w-1.5 before:rounded-full",
+                                            "flex items-center cursor-pointer rounded-md",
+                                            "transition-colors duration-200 ease-in-out",
                                             link.href === pathname
-                                                ? "text-orange-500 before:bg-orange-500"
-                                                : link.project
-                                                  ? "text-cyan-500 before:hidden before:bg-cyan-500 hover:before:block"
-                                                  : "text-gray-500 before:hidden before:bg-gray-300 hover:text-gray-600 hover:before:block dark:text-gray-400 dark:before:bg-gray-700 dark:hover:text-gray-300"
+                                                ? "bg-orange-100 dark:bg-orange-900/20"
+                                                : "hover:bg-gray-100 dark:hover:bg-gray-800/50"
                                         )}
+                                        onClick={() => toggleTopic(link.href)}
                                     >
-                                        {link.title}
-                                    </Link>
+                                        <Link
+                                            href={link.href}
+                                            onClick={(e) => {
+                                                e.stopPropagation()
+                                                onLinkClick && onLinkClick(e)
+                                            }}
+                                            className={clsx(
+                                                "flex items-center font-medium w-full py-2 px-3 rounded-md",
+                                                link.href === pathname
+                                                    ? "text-orange-500"
+                                                    : "text-gray-700 hover:text-gray-900 dark:text-gray-300 dark:hover:text-white"
+                                            )}
+                                        >
+                                            {link.icon && getIcon(link.icon)}
+                                            {link.title}
+                                        </Link>
+                                        {link.children &&
+                                            link.children.length > 0 && (
+                                                <span
+                                                    className={clsx(
+                                                        "ml-auto mr-3 text-xs flex-shrink-0",
```

Comment:
the touch target to expand/collapse child sections is small on mobile and hard to press.
This leads to accidentally clicking the parent in most cases.
A padding around those icons could help